src/app/components/Rules.js

```diff
@@ -0,0 +1,32 @@
+import { rulesContent } from "../tools/constants";
+
+class Rules {
+  constructor(content, id) {
+    this.rulesContent = rulesContent;
+    this.render(content, id);
+  }
+  contentRender(content, id) {
+    const container = document.querySelector(`#${id}`);
+    const rulesHeader = document.createElement('div');
+    const icon = document.createElement('img');
+    icon.setAttribute('src', '../static/assets/ui/school.svg');
+    rulesHeader.appendChild(icon);
+    const ButtonTittle = document.createElement('span');
+    const text = rulesHeader.appendChild(ButtonTittle);
+    text.innerHTML = content;
+    container.appendChild(rulesHeader);
+    const modeRules = document.createElement('p');
+    container.appendChild(modeRules);
+    modeRules.textContent = this.rulesContent[0];
```

Comment:
Jest dobrze, ale jakbyś chciał coś jeszcze poprawić, to możemy to 0 przekazać z poziomu app.js i przypisać to do jakiejś zmiennej, wtedy tą zmienna też byśmy przekazali do MainMenu jako ostatni parametr, dzięki temu byśmy mieli jedno miejsce które byśmy musieli w przyszłości zmienić, albo drugi sposób moze nawet lepsza, tą zmienna wrzucić do pliku constatns.js i stamtąd ją pobierać

Reply:
rozwiązanie drugie mi się bardzo podoba - dorzuciłem je w kolejnym moim branchu z głównym komponentem - to już merguje, żeby nie wstrzymywać pracy